### PR TITLE
Add ethical and sharia investing fields to firm form

### PR DESCRIPTION
**Pull request by neoeno**
image

It's the last two on the end thar

### CODE REVIEW

**Comment by benbarnett** [comment]
:+1:

File: spec/support/self_service/firm_edit_page.rb [hunk 1]
```diff
@@ -47,6 +52,22 @@ def offers_free_initial_meeting?
       !!offers_free_initial_meeting_true.checked?
     end
 
+    def offers_ethical_investing=(choice)
+      send("offers_ethical_investing_#{choice}").set(true)
+    end
+
+    def offers_ethical_investing?
+      !!offers_ethical_investing_true.checked?
```
Comment (JiggyPete) [inline spec/support/self_service/firm_edit_page.rb]: I'm wondering why the need for the double bang?
Can `:checked?` return nil?

**Comment by urfolomeus** [comment]
:+1:

File: spec/support/self_service/firm_edit_page.rb (hunk 1 above)
Comment (neoeno) [inline spec/support/self_service/firm_edit_page.rb]: I was following @munckymagik's pattern here — just checked with him and apparently `#checked?` returns something unusual, and we wanted to check against true or false

Comment (JiggyPete) [inline spec/support/self_service/firm_edit_page.rb]: makes sense

**Comment by JiggyPete** [comment]
:+1:

**Pull request merged**